### PR TITLE
Add CI steps and fsxc settings to pack and publish the package on NuGet website

### PR DESCRIPTION
**Pull request by realmarv**
Supersedes PR https://github.com/nblockchain/fsx/pull/8

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        if: startsWith(github.ref, 'refs/tags/')
+        run: echo "version=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Pack the NuGet package
+        run: |
+          echo $version
+          dotnet pack $(pwd)"/fsxc/fsxc.fsproj" -p:PackageVersion=$version
+      - name: Publish on NuGet website
+        env:
+          NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master' && env.NUGET_API_KEY != null
```
Comment (knocte) [inline .github/workflows/publish.yml:24]: @realmarv if you check for github.ref == 'refs/heads/master' here only, then tags won't be pushed to nuget

Comment (realmarv) [reply .github/workflows/publish.yml]: Done

**Comment by knocte** [comment]
Did you test it with a tag? Push a tag in your fork to see.

**Comment by realmarv** [comment]
> Did you test it with a tag? Push a tag in your fork to see.

I did test it with 0.6.0.2

**Comment by realmarv** [comment]
> Did you test it with a tag? Push a tag in your fork to see.

I did test it with 0.6.0.2

**Comment by knocte** [comment]
> I did test it with 0.6.0.2

Ok cool! Can you now unlist all versions you have published in nuget.org? Given that they were just for testing.

**Comment by realmarv** [comment]
> > I did test it with 0.6.0.2
> 
> Ok cool! Can you now unlist all versions you have published in nuget.org? Given that they were just for testing.

Done

**Pull request merged**